test/scale-infra/README.md

```diff
@@ -0,0 +1,228 @@
+    kube-public          Active   29m
+    kube-node-lease      Active   29m
+    default              Active   29m
+    kubestellar-report   Active   28m
+    ```
+
+
+2. Create WEC hosting instances:
+    
+    Set the following variables:
+    ```console
+    CLUSTER_NAME="wec"
+    NUM_HOSTING_INSTANCES=2
+    EC2_INSTANCE_TYPE="t2.xlarge"
+    ```
+
+    Then, deploy the WEC infra:
+
+    ```bash
+    ./deploy_wec_infra.sh --cluster-name $CLUSTER_NAME --region $REGION --vpc-name $VPC --wecs-hosting-instances $NUM_HOSTING_INSTANCES --instances-type $EC2_INSTANCE_TYPE --aws-key-name $EC2_SSH_PUBLIC_KEY --arch $ARCH --ec2-image-id $EC2_AMI
+    ```
+
+    Use the flag `--wecs-hosting-instances` to specify the number of ec2 instances to be created to host the WECs. You must create the WEC hosting instances in the same region as the KS control plane hosting infra created a step 1 - multiple regions deployment is not supported at the moment.  
+
+    Upon completion of the script's execution, an Ansible inventory file containing the IP addresses of the ec2 WEC hosting instances will be generated in the present directory at `.data/hosts_wec`.
+
+3. Create WEC kind clusters:
+
+    a) Update the inventory of the WEC instances to include the IP address of a master node from the K8s cluster created in step 1 above:
+
+    First, find the required master's IP address info in the following file: `.data/hosts_core`.
+
+    Next, use the following command to see the content of the WEC ansible inventory:
+    ```bash
+    cat .data/hosts_wec
+    ```
+
+    Sample output: 
+    ```console
+    [masters]
+
+    [add_workers]
+    worker1 ansible_host=192.168.56.1
+    worker1 ansible_host=192.168.56.2
+    ```
+
+    Lastly, edit the contents of the file `.data/hosts_wec` to include the IP address and name of a master node in the [masters] section, for example:
+    ```console
+    [masters]
+    master1 ansible_host=192.168.56.20
+
+    [add_workers]
+    worker1 ansible_host=192.168.56.1
+    worker1 ansible_host=192.168.56.2
+    ```
+
+    b) Create WEC kind clusters and connect to KS Core cluster:
+
+    ```bash
+    ansible-playbook -i .data/hosts_wec deploy_ks_wec.yaml --ssh-common-args="-o StrictHostKeyChecking=no" -e "region=$REGION vpc_name=$VPC num_wecs=1"
+    ```
+
+    Use the input paramater `num_wecs` to specify the number of kind clusters to be created for each WEC Hosting Instances. The above command creates kind WEC clusters and connects them to the KubeStellar core cluster created in step 1. Furthermore, it attaches a [KWOK](https://github.com/kubernetes-sigs/kwok) fake node to each kind cluster. 
+
+    c) Access the WEC kind clusters from Ansible control machine:
+
+    A kubeconfig file is generated for each WEC hosting instances at the following directory `.data/wecs_config/`. For example:
+
+    ```bash
+    ls .data/wecs_config/
+    ``` 
+
+    Output:
+    ```console
+    192-168-56-1.conf	192-168-56-2.conf
+    ```
```

Comment:
Testing commit 4c8ed2aa507db13c4e25ba505abb0842313145d8 , I see no such thing.

```console
bash-3.2$ ls -l .data
total 112
-rw-r-----@ 1 mspreitz  staff   5656 Mar 14 00:52 admin.conf
-rw-r--r--  1 mspreitz  staff  18043 Mar 14 00:49 ec2_instances_core.json
-rw-r--r--  1 mspreitz  staff   6063 Mar 14 01:22 ec2_instances_wec.json
-rw-r--r--  1 mspreitz  staff    147 Mar 14 00:49 hosts_core
-rw-r--r--@ 1 mspreitz  staff    113 Mar 14 01:32 hosts_wec
-rw-r--r--@ 1 mspreitz  staff     79 Mar 14 01:30 hosts_wec~
-rw-r--r--  1 mspreitz  staff    252 Mar 14 00:48 us-east-2_mspreitz_vpc_helper_vars.yaml
-rw-r--r--  1 mspreitz  staff   1101 Mar 14 00:58 us-east-2_mspreitz_wec_join_command
```

Reply:
@MikeSpreitzer, I just pushed a commit with all the changes. Initially, I was just updating the README files without pushing the code changes. Now, all the code changes are committed.